src/akvo/flow_services/uploader.clj

```diff
@@ -16,9 +16,14 @@
   (:import java.io.File
            org.waterforpeople.mapping.dataexport.RawDataSpreadsheetImporter
            java.util.zip.ZipFile
-           java.net.URLEncoder)
+           java.net.URLEncoder
+           org.apache.poi.ss.usermodel.Cell
```

Comment:
we can compact this 3 lines in one with `[org.apache.poi.ss.usermodel Cell Row Sheet]`